iset.mm

```diff
@@ -43783,6 +43804,33 @@ arbitrary sets or real numbers will not be provable (without the law of
     ( ccom cdm crn cxp wrel wss relco relssdmrn ax-mp dmcoss rncoss mp2an sstri
     xpss12 ) ABCZQDZQEZFZBDZAEZFZQGQTHABIQJKRUAHSUBHTUCHABLABMRUASUBPNO $.
 
+  ${
+    cossxp2.r $e |- ( ph -> R C_ ( A X. B ) ) $.
+    cossxp2.s $e |- ( ph -> S C_ ( B X. C ) ) $.
+    $( The composed of two relations is a relation, with bounds on its domain
+       and codomain.  (Contributed by BJ, 10-Jul-2022.) $)
```

Comment:
composed -> composition

Reply:
I do not know... See the "PS regarding wording choices" above. What does a native speaker, like @jkingdon, think ? Is it too weird ?

Reply:
Definitely should be "composition" here. This is probably normal English usage but it certainly is standard mathematical usage. See for example "the composition of g and f" at https://en.wikipedia.org/wiki/Function_composition .